Content.Server/Fax/FaxSystem.cs

```diff
@@ -43,6 +43,12 @@ public sealed class FaxSystem : EntitySystem
 
     private const string PaperSlotId = "Paper";
 
+    /// <summary>
+    ///     The prototype ID to use for faxed or copied entities if we can't get one from
+    ///     the paper entity for whatever reason.
+    /// </summary>
+    private const string DefaultPaperPrototypeId = "Paper";
```

Comment:
```suggestion
    /// <summary>
    ///     The prototype ID to use for faxed or copied entities if we can't get one from
    ///     the paper entity for whatever reason.
    /// </summary>
    [ValidatePrototypeId<EntityPrototype>]
    private const string DefaultPaperPrototypeId = "Paper";
```

Reply:
Aha! What a handy attribute – I shall have to remember that.